src/components/Icons/HereWalletIcon.tsx

```diff
@@ -1,6 +1,6 @@
 import { Icon, type IconProps } from '@chakra-ui/react';
 
-export const TokenHereIcon = ({ ...props }: IconProps) => {
+export const HereWalletIcon = ({ ...props }: IconProps) => {
```

Comment:
I put `Token` prefix so that it's easier for devs when browsing through the icons (all token related can be sorted easily). What do you think of this?

Reply:
HERE is a wallet, not a token. If its a token then yea, i agree with you